proxy/nginx/lualib/router.lua

```diff
@@ -35,7 +34,7 @@ if not res or res == ngx.null then
     key = prefix .. ":routes:$wildcard"
     res, err = red:get(key)
     if not res or res == ngx.null then
-        return
+        return ngx.exit(ngx.HTTP_BAD_GATEWAY)
```

Comment:
These changes are for avoiding errors "[error] 19#19: *2 fallback could not be resolved (3: Host not found)"